core/include/detray/intersection/intersection.hpp

```diff
@@ -49,7 +49,6 @@ struct intersection2D {
 
     using transform3_type = algebra_t;
     using scalar_type = typename algebra_t::scalar_type;
-    using point3 = typename algebra_t::point3;
```

Comment:
sorry, but aren't we using the 3d intersection solution for the most of the navigation parts?

Reply:
I think we have been using 2D solutions. Some of mask types require 3D information for boundary check but they are not kept inside the intersection object

Reply:
This PR now uses 3d intersection solution